polynote-spark/src/test/scala/polynote/kernel/interpreter/python/PySparkInterpreterSpec.scala

```diff
@@ -200,20 +196,20 @@ class PySparkInterpreterSpec extends FreeSpec with InterpreterSpec with Matchers
             "py" -> "four"))
         }
       }
+    }
 
-      "should capture py4j exceptions" in {
-        initialize()
-        try {
-          assertOutput("spark.read.text(\"doesnotexist\")"){ case _ => }
-        } catch {
-          case err: Throwable =>
-            err shouldBe a[RuntimeException]
-            err.getMessage should include ("AnalysisException: 'Path does not exist:")
-            err.getCause shouldBe a[RuntimeException]
-            err.getCause.getMessage should include ("Py4JJavaError: An error occurred while calling")
-            err.getCause.getCause shouldBe a[AnalysisException]
-            err.getCause.getCause.getMessage should include ("Path does not exist:")
-        }
+    "should capture py4j exceptions" in {
```

Comment:
This test was just moved. It was previously nested (erroneously) within the above PYTHONPATH related tests.